### PR TITLE
Will/rasa additions

### PR DESCRIPTION
**Pull request by whnds**

### CODE REVIEW

File: rasa/actions/actions.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# This files contains your custom actions which can be used to run
```
Comment (armstrongsam25) [inline rasa/actions/actions.py:1]: Remove these comments

File: rasa/config.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    epochs: 100
+    constrain_similarities: true
+  - name: FallbackClassifier
+    threshold: 0.3
+    ambiguity_threshold: 0.1
+
+# Configuration for Rasa Core.
+# https://rasa.com/docs/rasa/core/policies/
+policies:
+# # No configuration for policies was provided. The following default policies were used to train your model.
```
Comment (armstrongsam25) [inline rasa/config.yml:36]: Remove these comments too

File: rasa/data/nlu.yml [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  examples: |
+    - are you a bot?
+    - are you a human?
+    - am I talking to a bot?
+    - am I talking to a human?
+- intent: start
+  examples: |
+    - start
+    - startcal
+    - startcal [10](time)
```
Comment (armstrongsam25) [inline rasa/data/nlu.yml:92]: Does this work for other times?

File: rasa/domain.yml [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+    - type: slot
+      name: isNewUser
+      value: false
+  - text: "Welcome! What is your name?"
+    condition:
+    - type: slot
+      name: isNewUser
+      value: true
+
+  utter_cheer_up:
```
Comment (armstrongsam25) [inline rasa/domain.yml:31]: remove this

File: src/main/java/fasting/Webapi/API.java [hunk 5]
```diff
@@ -65,6 +77,22 @@ public Response incomingText(MultivaluedMap<String, String> formParams) {
...
+                request.setEntity(params);
+                HttpResponse response = httpClient.execute(request);
+                int code = response.getStatusLine().getStatusCode();
+                if (code==200) { // response comes back in the form [{recipient-id:XXX, text:hello world}] as a bytestream
+                    String text = gson.fromJson(EntityUtils.toString(response.getEntity()), JsonArray.class).get(0).getAsJsonObject().get("text").getAsString();
+                    System.out.println(text);
+                } else {
+                    System.out.println("HTTP Error Code "+code);
+                }
+
```
Comment (armstrongsam25) [inline src/main/java/fasting/Webapi/API.java:95]: is the intent the 'text' variable?

**Review by armstrongsam25 — CHANGES_REQUESTED** [review]
Remove:
rasa/__init__.py
rasa/actions/____init____.py
rasa/actions/____pycache____/*

Read comments

File: rasa/data/nlu.yml (hunk 3 above)
Comment (whnds) [reply rasa/data/nlu.yml:92]: Yes, those are formatting examples to help Rasa recognize a time input

File: src/main/java/fasting/Webapi/API.java [hunk 6]
```diff
@@ -65,6 +79,26 @@ public Response incomingText(MultivaluedMap<String, String> formParams) {
...
+                request.setEntity(params);
+                HttpResponse response = httpClient.execute(request);
+                int code = response.getStatusLine().getStatusCode();
+                if (code==200) { // response comes back in the form [{recipient-id, text}] as a bytestream
+                    String text = gson.fromJson(EntityUtils.toString(response.getEntity()), JsonArray.class).get(0).getAsJsonObject().get("text").getAsString();
+                    System.out.println(text); // message to the user
+                    HttpGet intent_request = new HttpGet("http://localhost:5005/conversations/"+participantId+"/tracker");
+                    HttpResponse intent_response = httpClient.execute(intent_request);
+                    String intent = gson.fromJson(EntityUtils.toString(intent_response.getEntity()), JsonElement.class).getAsJsonObject().get("latest_message").getAsJsonObject().get("intent").getAsJsonObject().get("name").getAsString();
+                    System.out.println(intent); // intent of previous message
```
Comment (armstrongsam25) [inline src/main/java/fasting/Webapi/API.java:97]: logger.error()

**Review by armstrongsam25 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**